engines/scumm/gfx_gui.cpp

```diff
@@ -2928,6 +3328,11 @@ void ScummEngine_v6::setUpMainMenuControls() {
 }
 
 void ScummEngine::drawMainMenuControls() {
+	if (_game.platform == Common::kPlatformSegaCD) {
```

Comment:
For a moment, I was thinking about subclassing `ScummEngine` to `ScummEngineSEGA` and overriding these methods there.

Reply:
Uhm, interesting. I'll see what would the effort of that be in terms of moving stuff around to another subclass; it's possible that that would have to wait for a different PR or a commit external to this one.